packages/icons/src/SVG/api.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The new SVGs have fill="none" on them which is currently causing issues :(

Reply:
Is it just api.svg or are there others? Does this prevent all of them or just the effected ones?

Reply:
Others too: 
![image](https://user-images.githubusercontent.com/12938082/196701577-b647883f-16e5-4ba4-a055-3f2969469ebf.png)

Reply:
That's so strange, because the all previewed fine when I exported them and added them to Github. It looks like its all the new ones.

Reply:
It's because the svgs have a built in path fill but we remove that so we can style the icons via the theme. We currently don't remove the fill from the svg but we could.